src/Security/Voter/HandleCommandVoter.php

```diff
@@ -25,7 +25,7 @@ class HandleCommandVoter extends Voter
      *
      * @var array
      */
-    private $commandRoleMapping;
+    private $commandRoleMapping = [];
```

Comment:
Why did you add this? It's constructor argument so it's always initialized there or am I missing something?